app/lib/common/widget/MembersList.dart

```diff
@@ -0,0 +1,65 @@
+// ignore_for_file: prefer_const_constructors, require_trailing_commas
+
+import 'package:flutter/material.dart';
+
+class MembersList extends StatefulWidget {
+  const MembersList({
+    Key? key,
+    required this.name,
+    required this.isAdmin,
+  }) : super(key: key);
+
+  final String name;
+  final bool isAdmin;
+
+  @override
+  MembersListState createState() => MembersListState();
+}
+
+class MembersListState extends State<MembersList> {
+  static const TextStyle optionStyle =
```

Comment:
Same as mentioned in 'InviteListView' review.

Reply:
Noted